lib/ci-helper.ts

```diff
@@ -511,19 +511,24 @@ export class CIHelper {
                     comment.author} is not permitted to use GitGitGadget`);
             }
 
+            const pr = await this.github.getPRInfo(comment.prNumber);
+
+            if (!pr || !pr.author) {
+                throw new Error(`Could not determine basic PR info for ${
+                    pullRequestURL}`);
+            }
+
```

Comment:
As this is unlikely to happen, I don't know whether it is really necessary to add that error message. It's not like incorrect usage of GitGitGadget can cause this...

Since you already have the error message, though, I wonder whether we shouldn't just go with something like this:

```typescript
const getPRAuthor = () => {
    const pr = await this.github.getPRInfo(comment.prNumber);
    if (!pr || !pr.author) {
        throw new Error(`Could not determine basic PR info for ${pullRequestURL}`);
    }
    return pr.author;
}
```

and then use `await getPRAuthor()` in the places below?

Reply:
> As this is unlikely to happen, I don't know whether it is really necessary to add that error message. It's not like incorrect usage of GitGitGadget can cause this...

I will remove the check and error message.  I had just copied what was done for /submit (although pr was checked after pr.author was already used).  If there is a problem retrieving the PR info, an error will be thrown.

Reply:
Okay.